apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/CompiledGraphQL.kt

```diff
@@ -2,9 +2,10 @@
 
 package com.apollographql.apollo3.api
 
+import com.apollographql.apollo3.annotations.ApolloInternal
 import com.apollographql.apollo3.api.CompiledArgument.Companion.resolveVariables
-import com.apollographql.apollo3.api.internal.json.BufferedSinkJsonWriter
-import com.apollographql.apollo3.api.internal.json.Utils
+import com.apollographql.apollo3.api.json.BufferedSinkJsonWriter
+import com.apollographql.apollo3.api.json.internal.Utils
```

Comment:
Renaming from  `com.apollographql.apollo3.api.internal.json` to `com.apollographql.apollo3.api.json.internal` to locate internal classes closer to where they are used.